### PR TITLE
aligned numeric Datatable Items right

### PR DESCRIPTION
**Pull request by hobbitronics**
- The Header Item as well as Row Item needs to have `numeric` to align right.
- Material handles alignment as long as we do this.

### CODE REVIEW

**Review by RussellChamp — APPROVED** [review]
lgtm

File: src/components/RecentActivityTable.svelte [hunk 1]
```diff
@@ -85,7 +85,7 @@ const getFormattedClaimItemPremium = (claim: Claim): string => {
       {/if}
     {:else}
       <DataRow>
-        <RowItem colspan="7">
+        <RowItem colspan={7}>
```
Comment (forevermatt) [inline src/components/RecentActivityTable.svelte:88]: Was this change intentional?

Comment (hobbitronics) [reply src/components/RecentActivityTable.svelte:88]: yes, colspan takes a number
